cmd/mount.go

```diff
@@ -643,6 +643,16 @@ func mount(c *cli.Context) error {
 			}
 		}
 		if blob != nil {
+			// test storage at startup to fail fast instead of throwing EIO in the middle of user's workload
+			if c.Bool("check-storage") && !metaConf.ReadOnly && !metaConf.NoBGJob {
```

Comment:
we can remove `metaConf.NoBGJob`

Also, we should test Get for read-only

Reply:
How about just removing those two checks `!metaConf.ReadOnly && !metaConf.NoBGJob`?

In `doTesting`, `store.Get` depends on `store.Put`, and `store.Delete` is always required (to do cleanup). It's not easy to just run one capability.

Reply:
Sounds good

Reply:
Updated